### ISSUE TITLE
Drag body no view other player in group

### ISSUE THREAD

**Issue by odinsvobik** [Crito-VanaheimServers/Working-Drag-Dead-Body-Script-For-Exile#1]
The drawn body is seen only by the player who pulls the body. Other players in the party does not see it and it can not heal.
But it's still great and thank you

**Comment by Crito-VanaheimServers** [comment]
Yes I have came across these issues as well. These were things that I was not able to experience in testing by myself so I didn't know about them until I put it into my live server and started playing with it on other players instead of my dead corpse lol.

**Comment by Spartanius1978** [comment]
Hello, the body after executing the throw command, the body still remains stuck to the player))))) How can I finish executing a script command?
